### ISSUE TITLE
After specific changes with Security Schemes apicurito will freeze

### ISSUE THREAD

**Issue by mmajerni** [Apicurio/apicurito#28]
Steps to reproduce: 

1. Create 3 security schemes each with a different type (e.g. scheme "a" with type basic, "b" with type apiKey and "c" with type oauth2). Fill just name and set security type. ("b" and "c" have warnings)

2. Create Security Requirements with these three schemes.
 
3. Edit second and third schemes and set security types to the Basic.

4. Again edit second and third schemes and set security types back. ("b" to apiKey and "c" to oauth2)

**WARNINGS are gone and it looks that schemes are good.** 
Now try to create anything (e.g. Path, Tag ...). It is not working. 

EDIT:
Problem with warnings --> API Key and Oauth2
Problem with freezing --> just Oauth2

**Comment by EricWittmann** [comment]
This has been fixed in upstream apicurio-studio.  Once a release is done and Apicurito is updated to use the latest editor component, the fix will flow here as well.

Note that I haven't yet fixed the problem of the disappearing Warning for API Key style security schemes.  I've opened a separate issue for that:  https://github.com/Apicurio/apicurio-studio/issues/689
